assets/js/vue/components/search/SearchForm.vue

```diff
@@ -227,7 +227,8 @@
                 fundingOrg: '',
                 researchGroup: '',
                 fundingCycle: '',
-                perPage: 10
+                perPage: 10,
+                projectDirector: ''
```

Comment:
Does this do anything here?

Reply:
This is part of the project director facet ticket